src/assets/themes/colors.jsx

```diff
@@ -30,4 +30,8 @@ export const TITLES_TEXT_COLOR = color4;
 export const BUTTON_LIGHT_TEXT_COLOR = color5;
 export const AWARD_POINTS_BACKGROUND = accentPoints;
 export const CHECKED_TOGGLER = checkedToggler;
-export const UNCHECKED_TOGGLER = uncheckedToggler;
\ No newline at end of file
+export const UNCHECKED_TOGGLER = uncheckedToggler;
+
+export const MAIN_PAGE_BACKGROUND_GENERAL = '#FAFAFA';
```

Comment:
It will be good if you move this color code into a variable (as I see this color already exists and is named "color5")